src/test/java/core/basesyntax/service/impl/FileReaderServiceTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceTest {
+    private static FileReaderService fileReaderService;
+    private static final String VALID_FILE_PATH = "src/test/resources/transaction.csv";
+
+    @BeforeClass
+    public static void beforeClass() {

```

Comment:
```suggestion
    public static void init() {
```
the same for other test classes